lib/rubocop/options.rb

```diff
@@ -9,6 +9,9 @@ class OptionArgumentError < StandardError; end
 
   # This class handles command line options.
   class Options
+    E_STDIN_NO_PATH = '-s/--stdin requires exactly one path, so that rubocop ' \
```

Comment:
I wonder if longer messages shouldn't link to the documentation instead.

Reply:
I still do not get it, what the hell it means. `'--stdin' '-' `or `'--stdin' 'xxx.rb'` product the same error. this improved msg does not help much:
```
['ruby', '-S', 'rubocop', '--format', 'emacs', '--force-exclusion', '--stdin', '/Users/admin/Downloads/rubocop-master/lib/rubocop/file_finder.rb']
SublimeLinter: WARNING: rubocop output:
-s/--stdin requires exactly one path.
```

Reply:
> I wonder if longer messages shouldn't link to the documentation instead.

I think, for a few dozen words, it is convenient to have it directly in the error message. If it were a few hundred words, I'd move it to the docs. That's just my opinion. 

Pro: docs can have formatting and code examples
Con: links can break in the future

> I still do not get it, what the hell it means. '--stdin' '-' or '--stdin' 'xxx.rb' product the same error. this improved msg does not help much:

Well, it will be compared against `Include` rules, so I think it's supposed to be a path, relative to the root of the project, right? Maybe we should also say that? "relative to the root of the project"